packages/charts/src/DualAxes/index.md

```diff
@@ -34,7 +34,7 @@ nav:
 
 | 参数        | 说明           | 类型      | 默认值 | 版本 |
 | :---------- | :------------- | :-------- | :----- | :--- |
-| geometry    | 图形类型       | string    | 'line' | -    |
+| geometry    | 图形类型       | string    | line | -    |
```

Comment:
- Suggest to highlight `enum` and `specific` value with `` like antd.

![image](https://github.com/oceanbase/oceanbase-design/assets/14918822/b39a786a-8134-41ef-9e39-a300694e9fb0)

Reply:
@wzc520pyfm Merge this PR first.

Reply:
I will fix that in next PR.